elasticsearch/docker/templates/Dockerfile.j2

```diff
@@ -108,8 +108,10 @@ RUN for plugin_path in `cat /tmp/plugins/plugins.list`; do \
       elasticsearch-plugin install --batch file:/tmp/plugins/$plugin_path; \
     done
 
-# Pull performance analyzer agent(RCA) from the plugin directory out into ESHOME
-RUN if [ -d plugins/opendistro_performance_analyzer/performance-analyzer-rca ]; then mv plugins/opendistro_performance_analyzer/performance-analyzer-rca ./; fi
+# Pull performance analyzer agent(RCA) from the plugin directory out into ESHOME and make the agent executable.
+RUN if [ -d plugins/opendistro_performance_analyzer/performance-analyzer-rca ]; then \
+     mv plugins/opendistro_performance_analyzer/performance-analyzer-rca ./ && \
+     chmod 755 performance-analyzer-rca/pa_bin/performance-analyzer-agent; fi
```

Comment:
If we start to use performance-analyzer-cli for all distros, do we still need to change permission on agent?

Reply:
I tried to change references to performance-analyzer-agent to point to performance-analyzer-cli, but we seemed to need a little more work to get it done correctly.

I created an [issue](https://github.com/opendistro-for-elasticsearch/performance-analyzer-rca/issues/263) in our RCA repository to consolidate them. We hope to address this by the next release.

Reply:
Sure.